hosted-cluster-config-operator/operator/config.go

```diff
@@ -64,7 +65,9 @@ func (c *HostedClusterConfigOperatorConfig) Scheme() *runtime.Scheme {
 func (c *HostedClusterConfigOperatorConfig) Manager() ctrl.Manager {
 	if c.manager == nil {
 		var err error
-		c.manager, err = ctrl.NewManager(c.TargetConfig(), ctrl.Options{
```

Comment:
can we do the same for `func (c *HostedClusterConfigOperatorConfig) KubeClient()`?

Reply:
@enxebre do you mean using `c.TargetConfig()` instead of `c.Config()` [here](https://github.com/openshift/hypershift/blob/c4397eb267138a5c80db0d1ec9e6eda1a4b45c2a/hosted-cluster-config-operator/operator/config.go#L166)?

Reply:
I mean setting the user agent in the KubeClient which uses s.config as well